app/tests/spec/views/complete_reset_password.js

```diff
@@ -361,9 +364,12 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
                   EMAIL,
                   PASSWORD,
                   relier,
+                  // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+                  /* eslint-disable */
```

Comment:
I kind of like the former option, @pdehaan. It wouldn't scale for larger objects but would work fine for these tests.